docs/environment_setup/JSON_pipeline_files.md

```diff
@@ -0,0 +1,34 @@
+# Data Factory - Testing Framework
+
+## JSON Files
+
+Ideally, you would integrate your project with a repository such as GIT. Once this is done you will have the JSON files for the Pipelines. If you need extra guidance on that the Microsoft Documentation provides very detailed information how to do it:
+
+Fabric
+[Git integration process - Microsoft Fabric | Microsoft Learn](https://learn.microsoft.com/en-us/fabric/cicd/git-integration/git-integration-process)
```

Comment:
As a best practice, we normally remove the localization from the link so we are inclusive of all locations.